cb_defense_syslog.py

```diff
@@ -272,6 +94,9 @@ def send_syslog_tls(server_url, port, data, output_type, output_format, ssl_veri
 
             client_socket.connect((server_url, port))
             client_socket.send(data.encode("utf-8"))
+            #
+            # if audit_data is not None:
+            #     client_socket.send(audit_data.encode("utf-8"))
```

Comment:
Any commented out code like this should either be deleted or used. "Use it or lose it" mentality 😄

Reply:
Commented out code has been deleted! Will be updated in next push.